mlflow/paddle/__init__.py

```diff
@@ -430,3 +444,190 @@ def predict(self, data):
 def _contains_pdparams(path):
     file_list = os.listdir(path)
     return any(".pdparams" in file for file in file_list)
+
+
+@experimental
+@autologging_integration(FLAVOR_NAME)
+def autolog(
```

Comment:
We recently refactored `autolog` for pytorch:
https://github.com/mlflow/mlflow/blob/master/mlflow/pytorch/_pytorch_autolog.py

This may give you some ideas for refactoring your code.

Reply:
Sure! Working on it!